packages/vaadin-upload/src/vaadin-upload.js

```diff
@@ -582,7 +582,7 @@ class UploadElement extends ElementMixin(ThemableMixin(PolymerElement)) {
     }
 
     const ini = Date.now();
-    const xhr = (file.xhr = this._createXhr(file));
+    const xhr = (file.xhr = this._createXhr());
```

Comment:
Unrelated, but this was easy to fix. The `_createXhr` method does not accept a parameter.